kf_lib_data_ingest/common/io.py

```diff
@@ -0,0 +1,169 @@
+"""
+Contains file readers (file -> dataframe) for file types that benefit from
```

Comment:
```suggestion
Contains file readers for file types that benefit from
```
Not all of the methods are for reading into a DataFrame

Reply:
√